### PR TITLE
feat: import lists from excel (DSP-1341)

### PR DESCRIPTION
**Pull request by lrosenth**
Added support for hierarchical lists in excel files and updated the documentation to make it more consistent and a bit easier to navigate.

### CODE REVIEW

**Comment by BalduinLandolt** [comment]
@lrosenth please add `knora/.DS_Store`to `.gitignore`

File: knora/bitstreams/test.csv [hunk 1]
```diff
@@ -1 +1,385 @@
...
Writings and drawings;Other Inscriptions;Labels;;;
Writings and drawings;Other drawings;;;;
Writings and drawings;Dockets;;;;
Writings and drawings;Mummy tags;;;;
Writings and drawings;Name stones;;;;
Writings and drawings;Book bindings;;;;
Writings and drawings;Work marks;;;;
Writings and drawings;Newspapers;;;;
Writings and drawings;Books;;;;
Writings and drawings;Documents;;;;
```
Comment (BalduinLandolt) [inline knora/bitstreams/test.csv:1]: This file really shouldn't be on github!

File: knora/onto-visualize.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import pygraphviz as pgv
```
Comment (BalduinLandolt) [inline knora/onto-visualize.py:1]: Is it possible you added stuff that was _not_ within the scope of the PR? ;-)

File: knora/tmp_xlsx2list.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
+import argparse
```
Comment (BalduinLandolt) [inline knora/tmp_xlsx2list.py:1]: Same code is in `onto_commons.py`. Is this temporary file still needed?

**Review by BalduinLandolt — CHANGES_REQUESTED** [review]
See my review comments.

Also note the following things I could not comment directly into the files:
 - please add `.DS_Store` to `.gitignore`
 - please ensure not to include any sensitive data in CSVs
 - be sure not to include any copyright protected files as test data (I haven't checked images and audio, but PDF is an NZZ article which is not public domain)

**Review by subotic — CHANGES_REQUESTED** [review]
- in general `BUILD` files should be named `BUILD.bazel`
- could you please clean-up a bit the `/knora` folder and move the test data to another directory. It has become very messy. Only data that needs to be shipped, should be part of the `/knora` folder.

File: knora/bitstreams/test.csv (hunk 1 above)
Comment (lrosenth) [reply knora/bitstreams/test.csv:1]: Replaced by a own cdv-file

File: knora/tmp_xlsx2list.py (hunk 3 above)
Comment (lrosenth) [reply knora/tmp_xlsx2list.py]: File removed – no longer needed after pull request

File: docs/index.md [hunk 4]
```diff
@@ -1,21 +1,29 @@
 [![PyPI version](https://badge.fury.io/py/dsp-tools.svg)](https://badge.fury.io/py/dsp-tools)
 
 # dsp-tools
-dsp-tools is a python package containing
+In order to work with a DaSCH Service Platform server (DSP-server), the user has to create a data
```
[the remark this answers is not in the document]
Comment (lrosenth) [reply docs/index.md:4]: Yeah

File: knora/onto-visualize.py (hunk 2 above)
Comment (lrosenth) [reply knora/onto-visualize.py]: Vorläufig wegkopiert :-)

**Review by subotic — APPROVED** [review]
great!

File: knora/bitstreams/test.csv (hunk 1 above)
Comment (lrosenth) [reply knora/bitstreams/test.csv:1]: No copyrighted data in PDF and images: all my stuff now

**Review by BalduinLandolt — APPROVED** [review]
Looks good to me

**Pull request merged**